static/app/views/codecov/tests/testAnalyticsTable/sortableHeader.tsx

```diff
@@ -47,22 +74,26 @@ function SortableHeader({fieldName, label, sort, tooltip, alignment}: HeaderPara
       >
         {label} {sort?.field === fieldName && sortArrow}
       </StyledLink>
+      {fieldName === 'testName' ? <WrapToggle /> : null}
```

Comment:
Thoughts on changing this from checking the field name to something like `enableWrapToggle`, and adding as a prop to the component?